scripts/replay-verify/replay_verify.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+import subprocess
+import shutil
+from multiprocessing import Pool, freeze_support
+from typing import IO, Tuple
+
+# This script runs the replay-verify from the root of aptos-core
+# It assumes the aptos-backup-cli, replay-verify, and db-backup binaries are already built with the release profile
+
+
+def replay_verify_partition(
+    n: int,
+    N: int,
+    history_start: int,
+    per_partition: int,
+    latest_version: int,
+    txns_to_skip: Tuple[int],
+    backup_config_template_path: str,
+):
+    """
+    Run replay-verify for a partition of the backup
+
+    n: partition number
+    N: total number of partitions
+    history_start: start version of the history
+    per_partition: number of versions per partition
+    latest_version: latest version in the backup
+    txns_to_skip: list of transactions to skip
+    backup_config_template_path: path to the backup config template
+    """
+    end = history_start + n * per_partition
+    if n == N - 1 and end < latest_version:
+        end = latest_version
+
+    start = end - per_partition
+    partition_name = f"run_{n}_{start}_{end}"
+
+    print(f"[partition {n}] spawning {partition_name}")
+    os.mkdir(partition_name)
+    shutil.copytree("metadata-cache", f"{partition_name}/metadata-cache")
```

Comment:
Shall we always clean these folders? otherwise it can leak disk space, because the partition names change over time, as more transaction versions show up in the backup.  Also what's the deal with overwriting existing folders? shall we set `dirs_exist_ok=true` if the intention is to reuse them?

Reply:
yea let's just always clean up these folders. Actions will run in a clean state though, so this is only a concern when running locally. I've changed it such that they are always cleaned up, unless you specify `REUSE_BACKUP_ARTIFACTS=true`.